### ISSUE TITLE
Support for more highlight groups

### ISSUE THREAD

**Issue by mroavi** [jose-elias-alvarez/buftabline.nvim#9]
Would it be possible to support the same highlight groups that vim-buftabline offers? From their docs:

```vim
Custom group                 Default link         Meaning
BufTabLineCurrent            TabLineSel           Buffer shown in current window
BufTabLineActive             PmenuSel             Buffer shown in other window
BufTabLineHidden             TabLine              Buffer not currently visible
BufTabLineFill               TabLineFill          Empty area
BufTabLineModifiedCurrent    BufTabLineCurrent    (Same as linked but 'modified')
BufTabLineModifiedActive     BufTabLineActive     (Same as linked but 'modified')
BufTabLineModifiedHidden     BufTabLineHidden     (Same as linked but 'modified')
```

**Comment by jose-elias-alvarez** [comment]
Sure, I'm planning a big rewrite of the current code base and can definitely work this in.

**Comment by jose-elias-alvarez** [comment]
Closed by #16.
